cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsTest.java

```diff
@@ -336,88 +335,44 @@ void exm125_careGaps_byIdentifier() {
                 .measureReportTypeIndividual();
     }
 
-    @Test
-    void exm125_careGaps_error_notSupportedProgram() {
```

Comment:
Question:  are you removing these tests because you removed the corresponding parameters from the production code?

Reply:
correct, these are no longer passable to the operation and therefore are not needed test scenarios